### ISSUE TITLE
❌ 🧩 🐘 Style tab of the Icon Box and Image Box widgets freezes inside the editor

### ISSUE THREAD

**Issue by JackMiles13** [elementor/elementor#25706]
### Prerequisites

- [X] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [X] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

[ bug elementor free Version 3.20.4]
Wordpress : 6.5.2 - Theme : Astra 4.6.11 (child theme activated)+ Woocommerce 8.7.0 - Php : 8.3.3
Hello,
I'm creating a new site and i noticed a bug after updating elementor from version 3.20.2 to 3.20.4 around the icon-box and image-box. I cannot style content, when I go to the tab and I click the dropdown icon to stylise the content, this part disapear and freeze the page. I switched back to V3.20.2 and those widgets work fine.

### Steps to reproduce

1-insert icon-box-content or image-box-content on a normal page
2- set svg, title and content (own svg or from library it's the same)
3- Go to style tab and click on "content" for stylization
4-Dropdown disapear and page is frozen

### Expected behavior

The drop-down menu for stylising the title or content should be accessible

### Isolating the problem

- [ ] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [ ] This bug happens with the Hello Elementor theme active.
- [X] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

```txt
Php : 8.3.3
Wordpress : 6.5.2
Theme : Astra 4.6.11 (child theme activated)
Woocommerce 8.7.0
Custom Fonts : 2.1.4
ShopLentor : 2.8.5
Admin and Site Enhancements (ASE) : 6.9.7
```

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by JackMiles13** [comment]
I just tested again Version 3.20.4 with PHP 8.2.16 and there's no issues with this PHP version. The trouble come from PHP 8.3

**Comment by toineenzo** [comment]
> I just tested again Version 3.20.4 with PHP 8.2.16 and there's no issues with this PHP version. The trouble come from PHP 8.3

That's because PHP 8.3 isn't officially supported yet with Elementor ;)

**Comment by nicholaszein** [comment]
Hello @JackMiles13.

Despite you not providing the required information as explained in the Bug Report, including following the instructions to Troubleshoot the issue, Isolate the Problem, and provide the full and required Elementor System Info, we did find a compatibility problem with these widgets that was fixed in version 3.21.2.

However, PHP 8.3 is still not supported by Elementor, nor by WordPress for that matter. We recommend using either PHP 8.0, or at maximum 8.1 for a stable experience. We are constantly improving our codebase to add support to further versions, but this also means introducing breaking changes, which need to be addressed cautiously.

Please, next time provide the required information.

Thank you
